api/api/templates/robots.txt

```diff
@@ -0,0 +1,8 @@
+# Block API endpoints
+User-agent: *
+Disallow: /v1/images/
+Disallow: /v1/audio/
+Disallow: /v1/auth/
+
+User-agent: GPTBot
+Disallow: /
```

Comment:
This bit is not necessary because we've already restricted all API endpoints to all bots (`*` on line 2). But we can keep it, no problems.